assets/src/edit-story/components/canvas/eyedropperLayer.js

```diff
@@ -0,0 +1,259 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useEffect, useRef } from 'react';
+import styled from 'styled-components';
+import { rgba, readableColor } from 'polished';
+import { FULLBLEED_RATIO } from '@web-stories-wp/units';
+
+/**
+ * Internal dependencies
+ */
+import { useCanvas, useLayout } from '../../app';
+import { useFocusOut, useKeyDownEffect } from '../../../design-system';
+import { Layer, PageArea } from './layout';
+import getColorFromPixelData from './utils/getColorFromPixelData';
+
+const EyedropperBackground = styled(Layer)`
+  background: rgba(0, 0, 0, 0.7);
+  cursor: not-allowed;
+  z-index: 2;
+`;
+
+const DisplayPageArea = styled(PageArea)`
+  position: absolute;
+`;
+
+const EyedropperCanvas = styled.div`
+  background: transparent;
+  border-radius: 5px;
+  color: black;
```

Comment:
Let's use the theme colours. What's the colour applied to here btw?

Reply:
My bad, that is unnecessary, was used for debugging.